ehrql/query_engines/base_sql.py

```diff
@@ -411,6 +411,25 @@ def get_sql_to_first_of_month(self, node):
     def to_first_of_month(self, date):
         raise NotImplementedError()
 
+    @get_sql.register(Function.MaximumOf)
+    def get_sql_maximum_of(self, node):
+        args = self.get_nary_function_args(node)
+        return self.get_aggregate_subquery(sqlalchemy.func.max, *args).label("greatest")
```

Comment:
What do the labels do here? Is this just about creating more readable SQL?

Reply:
yeah, just to give whoever might have to debug some SQL later a helping hand. Now that we're not following the convention of ANSI SQL `greatest` and `least` then these could maybe be changed to  something else helpful